scss/errorCard.module.scss

```diff
@@ -0,0 +1,41 @@
+@use './colors.module';
```

Comment:
Is is necessary to import the colors here? I don't really see it used in the file.